src/components/EncounterPreviewCard/EncounterPreviewCard.css

```diff
@@ -15,8 +14,6 @@
 
 .preview-section-title {
   color: #4AA3FC;
-  /* font-family: 'Merienda', cursive; */
-  /* font-family: 'Crimson Text', serif; */
   font-family: 'Arsenal', sans-serif;
```

Comment:
Adjusting these fonts maintains good readability and still keeps the same cohesive feel/theme in the app, nicely done!